scripts/run-http-api-with-fixtures-heartwood

```diff
@@ -0,0 +1,136 @@
+#!/bin/sh
```

Comment:
wanna change this file name to `run-httpd-with-fixtures`?

Reply:
02785d69f5252c6e4229d732c38f9e16c0ab427e